src/Domain/Infra/DependencyInjection/ExtensionHelper.php

```diff
@@ -70,7 +70,12 @@ public static function configureDoctrineOrm(ContainerBuilder $container, array $
         }
 
         $container->setParameter($param = 'msgphp.doctrine.type_config', $container->hasParameter($param) ? $typeConfig + $container->getParameter($param) : $typeConfig);
-        $container->setParameter($param = 'msgphp.doctrine.mapping_files', $container->hasParameter($param) ? array_merge($container->getParameter($param), $mappingFiles) : $mappingFiles);
```

Comment:
the ExtensionHelper runs _per_ bundle extension, so possibly multiple times. Here we only need to aggregate all bundle mappings in a single param, the rest of the config is global across all bundles.

So this can be reverted.

BundleHelper runs once, so here we can set the mapping_config param

```
setParameter(''msgphp.doctrine.mapping_config', [
  'mapping_dir' => '%kernel.project_dir%/config/packages/msgphp/docrine',
]);
```

Reply:
to be merged with the parameter already set by the user of course